src/Sentry.Unity/UnityLogException.cs

```diff
@@ -116,16 +125,19 @@ public SentryException ToSentryException()
                 }
 
                 var filenameWithoutZeroes = StripZeroes(filename);
-                frames.Add(new SentryStackFrame
+                var stackFrame = new SentryStackFrame
                 {
                     FileName = TryResolveFileNameForMono(filenameWithoutZeroes),
                     AbsolutePath = filenameWithoutZeroes,
                     Function = functionName,
-                    LineNumber = lineNo == -1 ? null : lineNo,
-                    InApp = functionName != null
-                        && !functionName.StartsWith("UnityEngine", StringComparison.Ordinal)
-                        && !functionName.StartsWith("System", StringComparison.Ordinal)
-                });
+                    LineNumber = lineNo == -1 ? null : lineNo
+                };
+                if (_options != null)

```

Comment:
```suggestion
                if (_options is not null)

```

Reply:
The test was validated by the previous tests where it should keep the same behavior by excluding UnityEngine and including BugFarm

https://github.com/getsentry/sentry-unity/blob/2c1dcbcbe0cb9bd8fcb82e0572cc6d9babf95496/test/Sentry.Unity.Tests/UnityLogExceptionTests.cs#L18-L27

Still I'll add an additional test case for validating mono.

Reply:
Add the mono and InAppInclude tests.
I don't think an InAppExclude test is required since that's already tested on ToSentryException_ParsingTestCases where Unity gets excluded.
https://github.com/getsentry/sentry-unity/blob/2c1dcbcbe0cb9bd8fcb82e0572cc6d9babf95496/src/Sentry.Unity/SentryUnityOptions.cs#L98-L99